src/base/interfaces/iapplication.h

```diff
@@ -30,11 +30,31 @@
 
 #pragma once
 
+#include <QtGlobal>
+#include <QObject>
```

Comment:
You don't really need to include entire QObject. QMetaObject is enough.

Reply:
Changed.